<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsAnimation</class>
 <widget class="QMainWindow" name="GraphicsAnimation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>776</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1234</width>
    <height>776</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1234</width>
    <height>776</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>小球碰撞动画</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(246, 255, 185);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1233</width>
      <height>611</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>666666</width>
      <height>66666</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>612</y>
      <width>621</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_stop">
       <property name="statusTip">
        <string>停止</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myimage.qrc">
         <normaloff>:/new/prefix1/image/stop.png</normaloff>:/new/prefix1/image/stop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_slower">
       <property name="statusTip">
        <string>减速一半</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myimage.qrc">
         <normaloff>:/new/prefix1/image/SubSpeed.png</normaloff>:/new/prefix1/image/SubSpeed.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>66</width>
         <height>66</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_begin">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="statusTip">
        <string>开始</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myimage.qrc">
         <normaloff>:/new/prefix1/image/Begin.png</normaloff>:/new/prefix1/image/Begin.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_faster">
       <property name="statusTip">
        <string>加速一倍</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myimage.qrc">
         <normaloff>:/new/prefix1/image/AddSpeed.png</normaloff>:/new/prefix1/image/AddSpeed.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>610</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pb_trace">
       <property name="statusTip">
        <string>轨迹跟踪</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myimage.qrc">
         <normaloff>:/new/prefix1/image/Trace.png</normaloff>:/new/prefix1/image/Trace.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_del_trace">
       <property name="statusTip">
        <string>清除轨迹</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myimage.qrc">
         <normaloff>:/new/prefix1/image/Clear.png</normaloff>:/new/prefix1/image/Clear.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_screenshots">
       <property name="statusTip">
        <string>截屏</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myimage.qrc">
         <normaloff>:/new/prefix1/image/ScreenShots.png</normaloff>:/new/prefix1/image/ScreenShots.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pb_like">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>620</y>
      <width>91</width>
      <height>63</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>加速一倍</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="myimage.qrc">
      <normaloff>:/new/prefix1/image/Like.png</normaloff>:/new/prefix1/image/Like.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1234</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>体型变换</string>
    </property>
    <addaction name="actionCircle"/>
    <addaction name="actionSquare"/>
    <addaction name="actionTriangle"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>记录此刻</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>时光倒流</string>
    </property>
    <addaction name="actionReverse"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>轨迹寻踪</string>
    </property>
    <addaction name="actionTrace"/>
    <addaction name="actionDelTrace"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionCircle"/>
   <addaction name="actionSquare"/>
   <addaction name="actionTriangle"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionDelTrace"/>
   <addaction name="separator"/>
   <addaction name="actionReverse"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCircle">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/new/prefix1/image/Circle.png</normaloff>:/new/prefix1/image/Circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
   <property name="statusTip">
    <string>圆形</string>
   </property>
  </action>
  <action name="actionSquare">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/new/prefix1/image/Square.png</normaloff>:/new/prefix1/image/Square.png</iconset>
   </property>
   <property name="text">
    <string>Square</string>
   </property>
   <property name="statusTip">
    <string>正方形</string>
   </property>
  </action>
  <action name="actionTriangle">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/new/prefix1/image/Triangle.png</normaloff>:/new/prefix1/image/Triangle.png</iconset>
   </property>
   <property name="text">
    <string>Triangle</string>
   </property>
   <property name="statusTip">
    <string>三角形</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/new/prefix1/image/ScreenShots.png</normaloff>:/new/prefix1/image/ScreenShots.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>截屏</string>
   </property>
  </action>
  <action name="actionReverse">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/new/prefix1/image/Reverse .png</normaloff>:/new/prefix1/image/Reverse .png</iconset>
   </property>
   <property name="text">
    <string>Reverse </string>
   </property>
   <property name="statusTip">
    <string>反向运动</string>
   </property>
  </action>
  <action name="actionTrace">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/new/prefix1/image/Trace.png</normaloff>:/new/prefix1/image/Trace.png</iconset>
   </property>
   <property name="text">
    <string>Trace</string>
   </property>
   <property name="statusTip">
    <string>轨迹跟踪</string>
   </property>
  </action>
  <action name="actionDelTrace">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/new/prefix1/image/Clear.png</normaloff>:/new/prefix1/image/Clear.png</iconset>
   </property>
   <property name="text">
    <string>DelTrace</string>
   </property>
   <property name="statusTip">
    <string>清除轨迹</string>
   </property>
  </action>
  <action name="actionStopTrace">
   <property name="icon">
    <iconset resource="myimage.qrc">
     <normaloff>:/new/prefix1/image/Back.png</normaloff>:/new/prefix1/image/Back.png</iconset>
   </property>
   <property name="text">
    <string>StopTrace</string>
   </property>
   <property name="statusTip">
    <string>放弃跟踪</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myimage.qrc"/>
 </resources>
 <connections/>
</ui>
